<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartmentView</class>
 <widget class="QWidget" name="DepartmentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DepartmentView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Currently Selected:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>User:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="usernameBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Message:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="messageBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Urgency:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="urgencyBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openDiscussionButton">
       <property name="text">
        <string>Open Discussion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeInquiryButton">
       <property name="text">
        <string>Close Inquiry</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
